src/main/java/com/github/krzysiekjodlowski/ox/FieldNumber.java

```diff
@@ -0,0 +1,18 @@
+package com.github.krzysiekjodlowski.ox;
+
+/**
+ * Represents each player move.
+ * @author Krzysztof Jodlowski
+ */
+class FieldNumber {
+    /**
+     * Field value.
+     */
+    private final int value;
+    /**
+     * @param value provided by client
+     */
+    FieldNumber(final int value) {
```

Comment:
One time you say symbol, other time you say value. Use one consistently

Reply:
What do you mean? Either in Symbol and FieldNumber value is a value:p